src/app/(app)/(home)/navbar-sidebar.tsx

```
@@ -0,0 +1,39 @@
+  return (
+    <Sheet>
+      <SheetTrigger>
+        <Button size={"icon"}>
+          <Menu />
+        </Button>
+      </SheetTrigger>
+      <SheetContent side="top">
+        {navbarItems.map((item) => (
+          <NavbarItem
+            href={item.href}
+            children={item.children}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)